data/human/human missions.txt

```diff
@@ -6849,3 +6849,320 @@ mission "FW Refugees to Humanika"
+			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`
+			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
+			`	You go back to the <ship>'s landing pad, and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`
+			choice
+				`	"I am Captain <first> <last>. Well met."`
+				`	"Hi, I'm <first>."`
+					goto loadup
+			`	They all frown at your formality.`
+
+			label loadup
+			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
+				accept
+
+			label out
+			`	"I'm sorry to hear that," she says between gulps of her smoothie. "I suppose it just wasn't meant to be." She finishes her "drink" with lightning speed and leaves.`
+				decline
+
+	on visit
+		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`
+
+
+mission "Adelita 3"
+	landing
+	name "Film slave transaction"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
+	source "Maker"
+	destination "Hippocrates"
+	"apparent payment" 200000
+	infiltrating
+	to offer
+		has "Adelita 2: done"
+	passengers 1
+	cargo "stealth drones" 1
+	on offer
+		conversation
+			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to take off for <planet>.`
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+
+mission "Adelita 4"
+	name "Slave trade"
+	description "Pick up the team on <destination>. (You will be paid <payment> after you complete the entire job.)"
+	source "Hippocrates"
+	destination "Maker"
+	landing
+	infiltrating
+	"apparent payment" 200000
+	to offer
+		has "Adelita 3: done"
+	passengers 1
+	cargo "stealth drones" 1
+
+	on offer
+		conversation
+			branch "player has a fleet"
+				"ships: Space Liner" + "ships: Transport" + "ships: Light Freighter" + "ships: Heavy Freighter" + "ships: Utility" + "ships: Interceptor" + "ships: Light Warship" + "ships: Medium Warship" + "ships: Heavy Warship" + "ships: Superheavy" > 1
```

Comment:
```suggestion
				"total ships" - ( "ships: Fighter" + "Ships: Drone" ) > 1
```

Reply:
If you had 3 ships and 2 fighters in your fleet, wouldn't it show the non-fleet message? Or is that acceptable to have to simplify this line?

Reply:
Fighters count towards the total ships count.

Reply:
Yeah, I'm just wondering if you could have a fleet of something like an Aerie, two Daggers, and two Ravens, the Daggers would cancel out the Ravens in the calculation (even though they're still in your fleet as you land) so the altered text about your fleet being in orbit wouldn't show?

Whereas if you just didn't count them there'd be no problem.

Reply:
If you had an Aerie, two Daggers and two Ravens, this condition would be calculated as follows:
`"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1`
`5 - ( 2 + 0 ) > 1`
`5 - 2 > 1`
`3 > 1`
and the branch will trigger to show the fleet text. The purpose of this check is to stop a solo carrier from being considered as a fleet, and having the fighters be left in orbit when they should be with the ship.